src/__tests__/lazyInlineRefResolver.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { lazyInlineRefResolver } from '../lazyInlineRefResolver';
```

Comment:
Obviously more tests will be added.
Http-spec had a decent coverage (I think) and all tests pass there + a few issues were fixed.